src/SearchField/README.md

```diff
@@ -0,0 +1,91 @@
+# SearchField
+
+Provides a basic search field component.
+
+## API
+
+### `screenReaderText` (string; required)
+`screenReaderText` specifies the text for screen readers within the `label` element of the `InputText` component.
+
+### `onSubmit` (function; required)
+`onSubmit` specifies a callback function for when the `SearchField` is submitted by the user. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+/>
+```
+
+### `onChange` (function; optional)
+`onChange` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+  onChange={(value) => { console.log(value); }}
+/>
+```
+
+### `onFocus` (function; optional)
+`onFocus` specifies a callback function for when the user focuses in the `SearchField` component. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+  onFocus={(value) => { console.log(value); }}
+/>
+```
+
+### `onBlur` (function; optional)
+`onBlur` specifies a callback function for when the user loses focus in the `SearchField` component. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+  onBlur={(value) => { console.log(value); }}
+/>
+```
+
+### `placeholder` (string; optional)
+`placeholder` specifies the placeholder text for the input. The default is an empty string.
+
+### `value` (string; optional)
+`value` specifies the initial value for the input. The default is an empty string.
+
+### `hasClearButton` (boolean; optional)
+`hasClearButton` specifies whether the search field has an additional button to allow the user to clear the `input` value. The default is `true`.
+
+### `roundedCorners` (string; optional)
+`roundedCorners` specifies the roundedness of the `SearchField` component. The options are `sm`, `md`, or `lg`.  The default is undefined, which results in no rounded corners.
```

Comment:
`roundedCorners` doesn't really imply anything about the "roundedness" of the component. If this was something like `roundedCornerSize` that might make more sense.